src/llmcompressor/transformers/utils/helpers.py

```diff
@@ -54,3 +59,223 @@ def detect_last_checkpoint(
             )
 
     return last_checkpoint
+
+
+def is_model_quantized_from_path(path: str):
+    """
+    Determine if model is quantized based on the config
+    """
+    config = AutoConfig.from_pretrained(path)
+    if config is not None:
+        if hasattr(config, "quantization_config"):
+            return True
+    return False
+
+
+def resolve_recipe(
+    model_path: Union[str, Path],
+    recipe: Union[str, Path, None] = None,
+) -> Union[str, None]:
+    """
+    Resolve the recipe to apply to the model.
+    :param recipe: the recipe to apply to the model.
```

Comment:
nit: new line b/w description and params

Reply:
ok